build/android-common.sh

```diff
@@ -206,7 +206,7 @@ get_android_arch() {
 }
 
 get_common_includes() {
-    echo "-I${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include -I${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/local/include"
+    echo "-isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include -isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include/${BUILD_HOST} -isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/local/include"
```

Comment:
This change causes [this error](https://ghostbin.co/paste/ukg2p) on Mac using NDK v21.0.6113669.

Reply:
The difference between our builds was that I had `API=24`, and you got `API=16` because you built with `--lts`.

Luckily, the latest releases of NDK don't need these common includes at all. They are handled by **llvm** via `--sysroot` parameter. In the light of that, I have updated my PR.

Please check that this works for you.

Reply:
It works for me too.